test/integration/test_integration.py

```diff
@@ -1186,17 +1186,55 @@ def test_autoquant_compile(self, device, dtype, m1, m2, k, n):
             torch.nn.ReLU(),
         ).to(device).to(dtype)
         example_input = torch.randn(m1, k, device=device, dtype=dtype)
-        example_input2 = torch.randn(m1, k, device=device, dtype=dtype)
+        example_input2 = torch.randn(m2, k, device=device, dtype=dtype)
         out = model(example_input)
 
-        mod = torchao.autoquant(torch.compile(model))
-        mod.forward_log_only(example_input)
+        mod = torchao.autoquant(torch.compile(model), manual=True)
+        mod(example_input)
         mod(example_input2)
+        mod.do_autoquant()
+
+        out2 = mod(example_input)
+        sqnr = SQNR(out, out2)
+        self.assertTrue(sqnr >= 30)
+
+    @parameterized.expand(COMMON_DEVICE_DTYPE)
+    @unittest.skipIf(not TORCH_VERSION_AFTER_2_3, "autoquant requires 2.3+.")
+    def test_autoquant_manual(self, device, dtype):
+        if device != "cuda" and dtype != torch.bfloat16:
```

Comment:
so fp16 is skipped? how come?

Reply:
removed this, i copied something from other tests at some point